packages/blade/src/components/Checkbox/CheckboxIcon/CheckboxIcon.tsx

```diff
@@ -8,27 +8,52 @@ import { useTheme } from '~components/BladeProvider';
 import Svg, { Path } from '~components/Icons/_Svg';
 import { getIn } from '~utils';
 
-const CheckedIcon = ({ color }: { color: string }) => {
+const svgSize = {
+  small: {
+    width: 8,
+    height: 8,
+  },
+  medium: {
+    width: 12,
+    height: 12,
+  },
+};
+
+const CheckedIcon = ({ color, size }: { color: string; size: 'small' | 'medium' }) => {
+  const width = `${svgSize[size].width}px`;
+  const height = `${svgSize[size].height}px`;
+
   return (
-    <Svg width="16px" height="16px" viewBox="0 0 16 16" fill="none">
+    <Svg width={width} height={height} viewBox="0 0 8 8" fill="none">
```

Comment:
**nit**: why not use `viewBox="0 0 24 24"`as the base since that's what we're doing for rest of the icons on blade too?

Reply:
I'll confirm first if the Check icon is same as the svg in checkbox. 

I remember saurav changed the svgs

Reply:
[Confirmed](https://razorpay.slack.com/archives/G01B3LQ9H0W/p1666863800967479?thread_ts=1666856828.448019&cid=G01B3LQ9H0W) with saurav. 

> You are not suppose to use icons inside checkboxes. You should export the SVG directly from the [Base DSL file](https://www.figma.com/file/LV9kjgzhkfsktvbRgTuGJ3/Blade---Base-DSL?node-id=13703%3A67331).